docs/web/sdk-language-strings.mdx

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
This feels quite similar to the Components => Internationalization page. I also prefer `Internationalization` over `Language Strings` for the title.

Reply:
Language Strings go farther than internationlization, and that's why I chose language strings over internationalization.

Language strings allow for string reuse, they allow for easy, user facing text updates that don't involve "real" code, and they simplify your component definition.

It is quite similiar to the components internationalization section, but registering and using language strings applies in both component code and in app config, so I felt it deserved it's own unifying article that is consumed by the [app config internationlization](https://deploy-preview-44--geocortex-dev-center.netlify.app/docs/web/concepts-advanced-app-config#internationalization) section and the [component internationalization section](https://deploy-preview-44--geocortex-dev-center.netlify.app/docs/web/sdk-components-internationalization).

However, this article **doesnt** link out to those ones, while those ones do link back to this one. I should probably add a "next steps" at the bottom linking out to using language strings in config and components.

Reply:
I'm okay keeping the name if we think it makes more sense, however i think this statement isn't quite correct:

> Language Strings go farther than internationlization

We've developed GXW including the creation of public APIs to support i18n. Language strings is a part of that entire initiative (i18n of the application). Other features would include things like number formatting, RTL, etc. 

The concept of i18n could easily expand in our docs to include best practices and document APIs for things like how to properly format a number using our APIs so it displays according to the end-user's device locale.

@eric-parsons any thoughts?

Reply:
While it's true that you can use language strings to override the text that ships with the product, the predominant use case for language strings is i18n. In custom code, there is no real reason to use language string other than i18n, since it just adds an unnecessary level of indirection. At the SDK level, that fact that the function to lookup values is called `translate()`, which reinforces this. And yes, eventually we'll need a place to document date/number formatting, time zones, RTL, and so forth. So I agree with Ian on this one: language strings ⊂ i18n, not the other way around.

Reply:
Changed the article to be titled the more generic "Internationalization" to allow for expansion with number formatting, time zones, RTL, etc. Added a "Next Steps" section at the bottom of this article linking to the references for internationalization in app config and components.